templates/admin/auth/edit.tmpl

```diff
@@ -114,10 +114,10 @@
 					<div class="inline field">
 						<div class="ui checkbox">
 							<label><strong>{{.locale.Tr "admin.auths.enable_ldap_groups"}}</strong></label>
-							<input type="checkbox" name="groups_enabled" class="js-ldap-group-toggle" {{if $cfg.GroupsEnabled}}checked{{end}}>
+							<input type="checkbox" name="groups_enabled" class="js-ldap-group-toggle"{{if $cfg.GroupsEnabled}} checked{{end}}>
```

Comment:
```suggestion
							<input type="checkbox" name="groups_enabled" class="js-ldap-group-toggle{{if $cfg.GroupsEnabled}} checked{{end}}">
```

The double quotes are in the wrong place here, and I'm not sure if it should be solved in this PR, or if it should be solved in a separate PR.